int32 room_id
string room_name
int32 status